modules/tinymce/src/core/main/ts/api/util/XHR.ts

```diff
@@ -59,10 +59,152 @@ const XHR: XHR = {
 
   /**
    * Sends a XMLHTTPRequest.
-   * Consult the TinyMCE Documentation for details on what settings this method takes.
    *
    * @method send
-   * @param {Object} settings Object will target URL, callbacks and other info needed to make the request.
+   * @param {Object} settings Object will target URL, callbacks and other info needed to make the request.<br /><br />
```

Comment:
Not sure if you're worried as this is what it was originally, but the first sentence is weird and I can't make sense of it.

Reply:
I honestly didn't read it. I was too focused on the chunk of missing info.

Reply:
oh, I think it was suppose to be s/will/with/

Reply:
Revised description in: https://github.com/tinymce/tinymce/pull/5924/commits/24059ba1645d967d6052ea2ddefc46dcf66f8310